azure-cosmosdb-table/setup.py

```diff
@@ -72,6 +72,7 @@
                          'cryptography',
                          'python-dateutil',
                          'requests',
-                     ] + (['futures'] if sys.version_info < (3, 0) else []),
+                         'futures; python_version<="2.7"'
```

Comment:
the earlier check eas broken since it pinned the requirement set of the python version used to make the universal wheel,